common/djangoapps/student/auth.py

```diff
@@ -61,6 +73,9 @@ def get_user_permissions(user, course_key, org=None):
     else:
         assert course_key is None
     all_perms = STUDIO_EDIT_ROLES | STUDIO_VIEW_USERS | STUDIO_EDIT_CONTENT | STUDIO_VIEW_CONTENT
+    # No one has studio permissions for CCX courses
+    if is_ccx_course(course_key):
+        return 0
```

Comment:
@haikuginger This block doesn't reference `all_perms`, so it might make sense to move it above the definition in L75.

Beyond that I'm wondering if it would be worth introducing a new constant (such as `STUDIO_ACCESS_DENIED`) instead of hard coding `0` here.